apis/core/lib/domain/column-mapping/create.ts

```diff
@@ -0,0 +1,34 @@
+import { GraphPointer } from 'clownface'
+import { rdf, schema } from '@tpluscode/rdf-ns-builders'
+import { cc } from '@cube-creator/core/namespace'
+import { ResourceStore } from '../../ResourceStore'
+import * as id from '../identifiers'
+import { resourceStore } from '../resources'
+import { NamedNode } from 'rdf-js'
+
+interface CreateColumnMappingCommand {
+  table: GraphPointer<NamedNode>
+  resource: GraphPointer
+  store?: ResourceStore
+}
+
+export async function createColumnMapping({
+  table,
+  resource,
+  store = resourceStore(),
+}: CreateColumnMappingCommand): Promise<GraphPointer> {
+  const label = resource.out(schema.label).term!
```

Comment:
Not sure we're going to have a label. We haven't put that in the sample data.

Reply:
I think it's required here: https://github.com/zazuko/cube-creator/blob/ff091d19307a0283134455aea97bd6c44b1851ce/apis/core/lib/domain/identifiers.ts#L30-L32

Reply:
It's using the `targetColumn` name. But I'm not even sure that was a good idea.